source/manual/troubleshoot-development-vm.html.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Troubleshoot your development VM
+parent: "/manual.html"
+layout: manual_layout
+section: Support
+owner_slack: "#govuk-developers"
+last_reviewed_on: 2017-08-08
+review_in: 3 months
+---
+
+Here are some things that can go wrong with the development VM.
+
+## Running out of disk space
+
+You may run out of disk space when replicating data into your development
+environment.
+
```

Comment:
It was related to running out of disk space, since there was no way to tell because the logging output was being swallowed.
https://github.com/alphagov/govuk-puppet/commit/78e1601397781c03f30158dc6243e6dd5570e9fa
It might be a bit too detailed for this page...